diff --git a/App/Views/Base.twig b/App/Views/Base.twig
index 5627991..57d7b23 100644
--- a/App/Views/Base.twig
+++ b/App/Views/Base.twig
@@ -66,9 +66,6 @@
     {% include "Includes/logon-modal.twig" %}
 {% endif %}
 
-{% if dev is defined %}
-    {% include "Includes/dev-info.twig" %}
-{% endif %}
 <script
         src="https://code.jquery.com/jquery-3.3.1.min.js"
         integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
diff --git a/App/Views/Includes/dev-info.twig b/App/Views/Includes/dev-info.twig
deleted file mode 100644
index 4d85733..0000000
--- a/App/Views/Includes/dev-info.twig
+++ /dev/null
@@ -1,13 +0,0 @@
-<div id="toogleDev" style="position: absolute; top: 150px; right: 0; background-color: rgba(51, 170, 51, .8);">
-    <p style="float: right" onclick="$('#dev-info').toggle();">Dev info</p>
-    <div id="dev-info" style="display: none; overflow:auto; height: 750px;">
-        {% for conf, val in dev_info %}
-
-            <p>{{ conf }}</p>
-            <pre style="background-color: #adb5bd; margin-left: 2em;">
-            {{ val|json_encode(constant('JSON_PRETTY_PRINT')) }}
-        </pre>
-
-        {% endfor %}
-    </div>
-</div>
\ No newline at end of file
diff --git a/Core/Controller.php b/Core/Controller.php
index d6e2a1e..2f9ee2e 100644
--- a/Core/Controller.php
+++ b/Core/Controller.php
@@ -119,27 +119,9 @@ abstract class Controller
         if ($this->alertBox->alertsPending()) {
             $this->data['alert_messages'] = $this->alertBox->getAlerts();
         }
-        if (Config::DEV_ENVIRONMENT) {
-            $this->devHelper();
-        }
+
         $twig = $this->container->getTemplate();
         $twig->display($template . '.twig', $this->data);
     }
 
-    protected function devHelper($var = '')
-    {
-        if (!isset($this->data['dev_info'])) {
-            $this->data['dev'] = true;
-            $classMethods = [];
-            if ($var != '') {
-                $classMethods['passed_var'] = $var;
-            }
-            $classMethods['class_object_methods'] = get_class_methods(get_class($this));
-            $classMethods['class_object_vars'] = get_object_vars($this);
-            $classMethods['session_vars'] = $_SESSION;
-
-            $this->data['dev_info'] = $classMethods;
-        }
-
-    }
 }
\ No newline at end of file
